triagebot.toml

```diff
@@ -0,0 +1,59 @@
+[assign]
+warn_non_default_branch = true
+# contributing_url = "https://rustc-dev-guide.rust-lang.org/contributing.html" # FIXME: configure
+
+[assign.adhoc_groups]
+# This is a special group that will be used if none of the `owners` entries matches.
+fallback = ["@WaffleLapkin", "@Hirrolot"]
+
+[assign.owners]
+"crates/teloxide-core" = ["@WaffleLapkin"]
+"crates/teloxide-macros" = ["@WaffleLapkin"]
+".github" = ["@WaffleLapkin"]
```

Comment:
@Hirrolot if you want to get assigned when specific files are edited, lmk.

Reply:
I think I should be notified when someone contributes to `crates/teloxide`.

Reply:
Do you want to be *notified* or *assigned*? Cause there are mechanisms for both

Reply:
Well, assigned.